drivers/volume/portworx/portworx.go

```diff
@@ -2809,6 +2840,24 @@ func getSDKContainerPort() (int32, error) {
 	return 0, fmt.Errorf("px-sdk target port not found in service")
 }
 
+func getImageList(pxVersion, k8sVersion string) (map[string]string, error) {
+	imageList := make(map[string]string)
+	url := fmt.Sprintf("https://install.portworx.com/%s/version?kbver=%s", pxVersion, k8sVersion)
+	stdout, err := exec.Command("curl", "-L", url).CombinedOutput()
```

Comment:
We should use "net/http" here instead of curl, see here how we call URL and format images from it https://github.com/portworx/spawn/blob/a7fdc207ca595947670d66606a1c19cddfa54984/cmd/provision_px_k8s.go#L944